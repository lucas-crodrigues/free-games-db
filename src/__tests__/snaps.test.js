import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import React from 'react';
import store from '../redux/configureStore';
import Home from '../pages/home';
import Games from '../pages/details';

describe('check snapshot of pages', () => {
  test('check snapshot of home', () => {
    const games = [{game: 'game1'}, {game: 'game2'}]
    const pcGames = [{game: 'game1'}, {game: 'game2'}]
    const consoleGames = [{game: 'game1'}, {game: 'game2'}]
    const mobileGames = [{game: 'game1'}, {game: 'game2'}]
    const allGames = [{game: 'game1'}, {game: 'game2'}]
    const allDLC = [{game: 'game1'}, {game: 'game2'}]
    const allEarly = [{game: 'game1'}, {game: 'game2'}]

    const component = render(
      <React.StrictMode>
        <Provider store={store}>
          <Router>
            <Home 
              key="games"
              games={games}
              pcGames={pcGames}
              consoleGames={consoleGames}
              mobileGames={mobileGames}
              allGames={allGames}
              allDLC={allDLC}
              allEarly={allEarly}            
            />
          </Router>
        </Provider>
      </React.StrictMode>,
    );
    expect(component).toMatchSnapshot();
  });  
  
  test('check snapshot of details', () => {
    const games = [
      {
      id: 1867,
      title: "Call of Duty: Warzone KF Weapon Charm Key Giveaway",
      worth: "N/A",
      thumbnail: "https://www.gamerpower.com/offers/1/630e76b5dd23c.jpg",
      image: "https://www.gamerpower.com/offers/1b/630e76b5dd23c.jpg",
      description: "Claim your free KF Weapon Charm Key for Call of Duty: Warzone and jump into the action in style!",
      instructions: "1. Download and Install the Steelseries GG software.\r\n2. Log in to the app, go to the \"Giveaway\" section and click the button to unlock your key.\r\n3. Follow the giveaway instructions to redeem your key.",
      open_giveaway_url: "https://www.gamerpower.com/open/call-of-duty-warzone-kf-weapon-charm-key-giveaway",
      published_date: "2022-08-30 22:44:37",
      type: "DLC",
      platforms: "PC",
      end_date: "N/A",
      users: 790,
      status: "Active",
      gamerpower_url: "https://www.gamerpower.com/call-of-duty-warzone-kf-weapon-charm-key-giveaway",
      open_giveaway: "https://www.gamerpower.com/open/call-of-duty-warzone-kf-weapon-charm-key-giveaway"
    },
    {
      id: 1866,
      title: "Dead by Daylight: Free 100k Bloodpoints Key",
      worth: "N/A",
      thumbnail: "https://www.gamerpower.com/offers/1/630e74876116e.jpg",
      image: "https://www.gamerpower.com/offers/1b/630e74876116e.jpg",
      description: "Claim your free 100.000 Bloodpoints key for Dead by Daylight and jump into the action! You just need to install the Steelseries GG software to claim your key!",
      instructions: "1. Download and Install the Steelseries GG software.\r\n2. Log in to the app, go to the \"Giveaway\" section and click the button to unlock your key.\r\n3. Start the game and redeem the code in-game by going to the Store, clicking \"Redeem\" and enter your code.",
      open_giveaway_url: "https://www.gamerpower.com/open/dead-by-daylight-free-100k-bloodpoints-key",
      published_date: "2022-08-30 22:35:19",
      type: "DLC",
      platforms: "PC",
      end_date: "N/A",
      users: 600,
      status: "Active",
      gamerpower_url: "https://www.gamerpower.com/dead-by-daylight-free-100k-bloodpoints-key",
      open_giveaway: "https://www.gamerpower.com/open/dead-by-daylight-free-100k-bloodpoints-key"
    },
  ]

    const component = render(
      <React.StrictMode>
        <Provider store={store}>
          <Router>
            <Games key="gamesPage" gameList={games} icon={'gameimg'} title="All Content" />
          </Router>
        </Provider>
      </React.StrictMode>,
    );
    expect(component).toMatchSnapshot();
  });
})

