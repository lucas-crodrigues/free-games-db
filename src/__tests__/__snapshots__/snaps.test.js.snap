// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check snapshot of pages check snapshot of details 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="header"
      >
        <a
          aria-current="page"
          class="active"
          href="/"
        >
          ‹
        </a>
        <h1>
          Free Games DB
        </h1>
      </div>
      <div
        class="pageTitle"
      >
        <img
          alt="logo"
          src="gameimg"
        />
        <h2>
          All Content
        </h2>
      </div>
      <div
        class="filters"
      >
        <a
          class=""
          href="/games"
        >
          All Free Content
        </a>
        <a
          class=""
          href="/pc"
        >
          All PC Content
        </a>
        <a
          class=""
          href="/console"
        >
          All Console Content
        </a>
        <a
          class=""
          href="/mobile"
        >
          All Mobile Content
        </a>
      </div>
      <div
        class="gamesList"
      >
        <div
          class="gameCard"
        >
          <div
            class="gameTitle"
          >
            <a
              href="https://www.gamerpower.com/open/call-of-duty-warzone-kf-weapon-charm-key-giveaway"
              rel="noopener noreferrer"
              target="_blank"
            >
              <img
                alt="Game Call of Duty: Warzone KF Weapon Charm Key Giveaway"
                class="gamePic"
                src="https://www.gamerpower.com/offers/1b/630e76b5dd23c.jpg"
              />
              Call of Duty: Warzone KF Weapon Charm Key Giveaway
            </a>
            <p
              class="gameType"
              id="DLC"
            >
              DLC
            </p>
          </div>
          <details
            class="info"
          >
            <summary>
              More Info
            </summary>
            <p
              style="white-space: pre-wrap;"
            >
              1. Download and Install the Steelseries GG software.
2. Log in to the app, go to the "Giveaway" section and click the button to unlock your key.
3. Follow the giveaway instructions to redeem your key.
            </p>
            <small>
              Works with:
              PC
            </small>
            <small>
              Ends at:
              N/A
            </small>
          </details>
        </div>
        <div
          class="gameCard"
        >
          <div
            class="gameTitle"
          >
            <a
              href="https://www.gamerpower.com/open/dead-by-daylight-free-100k-bloodpoints-key"
              rel="noopener noreferrer"
              target="_blank"
            >
              <img
                alt="Game Dead by Daylight: Free 100k Bloodpoints Key"
                class="gamePic"
                src="https://www.gamerpower.com/offers/1b/630e74876116e.jpg"
              />
              Dead by Daylight: Free 100k Bloodpoints Key
            </a>
            <p
              class="gameType"
              id="DLC"
            >
              DLC
            </p>
          </div>
          <details
            class="info"
          >
            <summary>
              More Info
            </summary>
            <p
              style="white-space: pre-wrap;"
            >
              1. Download and Install the Steelseries GG software.
2. Log in to the app, go to the "Giveaway" section and click the button to unlock your key.
3. Start the game and redeem the code in-game by going to the Store, clicking "Redeem" and enter your code.
            </p>
            <small>
              Works with:
              PC
            </small>
            <small>
              Ends at:
              N/A
            </small>
          </details>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="header"
    >
      <a
        aria-current="page"
        class="active"
        href="/"
      >
        ‹
      </a>
      <h1>
        Free Games DB
      </h1>
    </div>
    <div
      class="pageTitle"
    >
      <img
        alt="logo"
        src="gameimg"
      />
      <h2>
        All Content
      </h2>
    </div>
    <div
      class="filters"
    >
      <a
        class=""
        href="/games"
      >
        All Free Content
      </a>
      <a
        class=""
        href="/pc"
      >
        All PC Content
      </a>
      <a
        class=""
        href="/console"
      >
        All Console Content
      </a>
      <a
        class=""
        href="/mobile"
      >
        All Mobile Content
      </a>
    </div>
    <div
      class="gamesList"
    >
      <div
        class="gameCard"
      >
        <div
          class="gameTitle"
        >
          <a
            href="https://www.gamerpower.com/open/call-of-duty-warzone-kf-weapon-charm-key-giveaway"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Game Call of Duty: Warzone KF Weapon Charm Key Giveaway"
              class="gamePic"
              src="https://www.gamerpower.com/offers/1b/630e76b5dd23c.jpg"
            />
            Call of Duty: Warzone KF Weapon Charm Key Giveaway
          </a>
          <p
            class="gameType"
            id="DLC"
          >
            DLC
          </p>
        </div>
        <details
          class="info"
        >
          <summary>
            More Info
          </summary>
          <p
            style="white-space: pre-wrap;"
          >
            1. Download and Install the Steelseries GG software.
2. Log in to the app, go to the "Giveaway" section and click the button to unlock your key.
3. Follow the giveaway instructions to redeem your key.
          </p>
          <small>
            Works with:
            PC
          </small>
          <small>
            Ends at:
            N/A
          </small>
        </details>
      </div>
      <div
        class="gameCard"
      >
        <div
          class="gameTitle"
        >
          <a
            href="https://www.gamerpower.com/open/dead-by-daylight-free-100k-bloodpoints-key"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Game Dead by Daylight: Free 100k Bloodpoints Key"
              class="gamePic"
              src="https://www.gamerpower.com/offers/1b/630e74876116e.jpg"
            />
            Dead by Daylight: Free 100k Bloodpoints Key
          </a>
          <p
            class="gameType"
            id="DLC"
          >
            DLC
          </p>
        </div>
        <details
          class="info"
        >
          <summary>
            More Info
          </summary>
          <p
            style="white-space: pre-wrap;"
          >
            1. Download and Install the Steelseries GG software.
2. Log in to the app, go to the "Giveaway" section and click the button to unlock your key.
3. Start the game and redeem the code in-game by going to the Store, clicking "Redeem" and enter your code.
          </p>
          <small>
            Works with:
            PC
          </small>
          <small>
            Ends at:
            N/A
          </small>
        </details>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`check snapshot of pages check snapshot of home 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="homePage"
      >
        <div
          class="header"
        >
          <div
            class="invisible"
          />
          <h1>
            Free Games DB
          </h1>
        </div>
        <div
          class="logo"
        >
          <img
            alt="logo"
            src="logo.svg"
          />
        </div>
        <div
          class="main"
        >
          <h3>
            STATS
          </h3>
          <div
            class="stats"
          >
            <div>
              <small>
                All
                <br />
                games
              </small>
              <p>
                2
              </p>
            </div>
            <div>
              <small>
                PC
                <br />
                games
              </small>
              <p>
                2
              </p>
            </div>
            <div>
              <small>
                Console
                <br />
                games
              </small>
              <p>
                2
              </p>
            </div>
            <div>
              <small>
                Mobile
                <br />
                games
              </small>
              <p>
                2
              </p>
            </div>
            <div>
              <small>
                Full
                <br />
                games
              </small>
              <p>
                2
              </p>
            </div>
            <div>
              <small>
                All
                <br />
                DLCs
              </small>
              <p>
                2
              </p>
            </div>
            <div>
              <small>
                Early
                <br />
                Access
              </small>
              <p>
                2
              </p>
            </div>
          </div>
        </div>
        <div
          class="pages"
        >
          <div
            class="gameContent page"
          >
            <a
              class="pageLink"
              href="/games"
            >
              All Free Content:
               
              2
            </a>
          </div>
          <div
            class="pcContent page"
          >
            <a
              class="pageLink"
              href="/pc"
            >
              All PC Content:
               
              2
            </a>
          </div>
          <div
            class="consoleContent page"
          >
            <a
              class="pageLink"
              href="/console"
            >
              All Console Content:
               
              2
            </a>
          </div>
          <div
            class="mobileContent page"
          >
            <a
              class="pageLink"
              href="/mobile"
            >
              All Mobile Content:
               
              2
            </a>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="homePage"
    >
      <div
        class="header"
      >
        <div
          class="invisible"
        />
        <h1>
          Free Games DB
        </h1>
      </div>
      <div
        class="logo"
      >
        <img
          alt="logo"
          src="logo.svg"
        />
      </div>
      <div
        class="main"
      >
        <h3>
          STATS
        </h3>
        <div
          class="stats"
        >
          <div>
            <small>
              All
              <br />
              games
            </small>
            <p>
              2
            </p>
          </div>
          <div>
            <small>
              PC
              <br />
              games
            </small>
            <p>
              2
            </p>
          </div>
          <div>
            <small>
              Console
              <br />
              games
            </small>
            <p>
              2
            </p>
          </div>
          <div>
            <small>
              Mobile
              <br />
              games
            </small>
            <p>
              2
            </p>
          </div>
          <div>
            <small>
              Full
              <br />
              games
            </small>
            <p>
              2
            </p>
          </div>
          <div>
            <small>
              All
              <br />
              DLCs
            </small>
            <p>
              2
            </p>
          </div>
          <div>
            <small>
              Early
              <br />
              Access
            </small>
            <p>
              2
            </p>
          </div>
        </div>
      </div>
      <div
        class="pages"
      >
        <div
          class="gameContent page"
        >
          <a
            class="pageLink"
            href="/games"
          >
            All Free Content:
             
            2
          </a>
        </div>
        <div
          class="pcContent page"
        >
          <a
            class="pageLink"
            href="/pc"
          >
            All PC Content:
             
            2
          </a>
        </div>
        <div
          class="consoleContent page"
        >
          <a
            class="pageLink"
            href="/console"
          >
            All Console Content:
             
            2
          </a>
        </div>
        <div
          class="mobileContent page"
        >
          <a
            class="pageLink"
            href="/mobile"
          >
            All Mobile Content:
             
            2
          </a>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`check snapshot of pages check snapshot of home 2`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="header"
      >
        <a
          aria-current="page"
          class="active"
          href="/"
        >
          ‹
        </a>
        <h1>
          Free Games DB
        </h1>
      </div>
      <div
        class="pageTitle"
      >
        <img
          alt="logo"
          src="gameimg"
        />
        <h2>
          All Content
        </h2>
      </div>
      <div
        class="filters"
      >
        <a
          class=""
          href="/games"
        >
          All Free Content
        </a>
        <a
          class=""
          href="/pc"
        >
          All PC Content
        </a>
        <a
          class=""
          href="/console"
        >
          All Console Content
        </a>
        <a
          class=""
          href="/mobile"
        >
          All Mobile Content
        </a>
      </div>
      <div
        class="gamesList"
      >
        <div
          class="gameCard"
        >
          <div
            class="gameTitle"
          >
            <a
              href="https://www.gamerpower.com/open/call-of-duty-warzone-kf-weapon-charm-key-giveaway"
              rel="noopener noreferrer"
              target="_blank"
            >
              <img
                alt="Game Call of Duty: Warzone KF Weapon Charm Key Giveaway"
                class="gamePic"
                src="https://www.gamerpower.com/offers/1b/630e76b5dd23c.jpg"
              />
              Call of Duty: Warzone KF Weapon Charm Key Giveaway
            </a>
            <p
              class="gameType"
              id="DLC"
            >
              DLC
            </p>
          </div>
          <details
            class="info"
          >
            <summary>
              More Info
            </summary>
            <p
              style="white-space: pre-wrap;"
            >
              1. Download and Install the Steelseries GG software.
2. Log in to the app, go to the "Giveaway" section and click the button to unlock your key.
3. Follow the giveaway instructions to redeem your key.
            </p>
            <small>
              Works with:
              PC
            </small>
            <small>
              Ends at:
              N/A
            </small>
          </details>
        </div>
        <div
          class="gameCard"
        >
          <div
            class="gameTitle"
          >
            <a
              href="https://www.gamerpower.com/open/dead-by-daylight-free-100k-bloodpoints-key"
              rel="noopener noreferrer"
              target="_blank"
            >
              <img
                alt="Game Dead by Daylight: Free 100k Bloodpoints Key"
                class="gamePic"
                src="https://www.gamerpower.com/offers/1b/630e74876116e.jpg"
              />
              Dead by Daylight: Free 100k Bloodpoints Key
            </a>
            <p
              class="gameType"
              id="DLC"
            >
              DLC
            </p>
          </div>
          <details
            class="info"
          >
            <summary>
              More Info
            </summary>
            <p
              style="white-space: pre-wrap;"
            >
              1. Download and Install the Steelseries GG software.
2. Log in to the app, go to the "Giveaway" section and click the button to unlock your key.
3. Start the game and redeem the code in-game by going to the Store, clicking "Redeem" and enter your code.
            </p>
            <small>
              Works with:
              PC
            </small>
            <small>
              Ends at:
              N/A
            </small>
          </details>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="header"
    >
      <a
        aria-current="page"
        class="active"
        href="/"
      >
        ‹
      </a>
      <h1>
        Free Games DB
      </h1>
    </div>
    <div
      class="pageTitle"
    >
      <img
        alt="logo"
        src="gameimg"
      />
      <h2>
        All Content
      </h2>
    </div>
    <div
      class="filters"
    >
      <a
        class=""
        href="/games"
      >
        All Free Content
      </a>
      <a
        class=""
        href="/pc"
      >
        All PC Content
      </a>
      <a
        class=""
        href="/console"
      >
        All Console Content
      </a>
      <a
        class=""
        href="/mobile"
      >
        All Mobile Content
      </a>
    </div>
    <div
      class="gamesList"
    >
      <div
        class="gameCard"
      >
        <div
          class="gameTitle"
        >
          <a
            href="https://www.gamerpower.com/open/call-of-duty-warzone-kf-weapon-charm-key-giveaway"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Game Call of Duty: Warzone KF Weapon Charm Key Giveaway"
              class="gamePic"
              src="https://www.gamerpower.com/offers/1b/630e76b5dd23c.jpg"
            />
            Call of Duty: Warzone KF Weapon Charm Key Giveaway
          </a>
          <p
            class="gameType"
            id="DLC"
          >
            DLC
          </p>
        </div>
        <details
          class="info"
        >
          <summary>
            More Info
          </summary>
          <p
            style="white-space: pre-wrap;"
          >
            1. Download and Install the Steelseries GG software.
2. Log in to the app, go to the "Giveaway" section and click the button to unlock your key.
3. Follow the giveaway instructions to redeem your key.
          </p>
          <small>
            Works with:
            PC
          </small>
          <small>
            Ends at:
            N/A
          </small>
        </details>
      </div>
      <div
        class="gameCard"
      >
        <div
          class="gameTitle"
        >
          <a
            href="https://www.gamerpower.com/open/dead-by-daylight-free-100k-bloodpoints-key"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Game Dead by Daylight: Free 100k Bloodpoints Key"
              class="gamePic"
              src="https://www.gamerpower.com/offers/1b/630e74876116e.jpg"
            />
            Dead by Daylight: Free 100k Bloodpoints Key
          </a>
          <p
            class="gameType"
            id="DLC"
          >
            DLC
          </p>
        </div>
        <details
          class="info"
        >
          <summary>
            More Info
          </summary>
          <p
            style="white-space: pre-wrap;"
          >
            1. Download and Install the Steelseries GG software.
2. Log in to the app, go to the "Giveaway" section and click the button to unlock your key.
3. Start the game and redeem the code in-game by going to the Store, clicking "Redeem" and enter your code.
          </p>
          <small>
            Works with:
            PC
          </small>
          <small>
            Ends at:
            N/A
          </small>
        </details>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
